lib/client.js

```diff
@@ -129,10 +132,12 @@ var clientOptions = require('./client-options');
  * </p>
  * Default: false.
```

Comment:
There are some query options that could be candidates for execution profile options as well, I think these are reasonable considerations:

* fetchSize
* customPayload
* retryOnTimeout
* traceQuery

The other ones seem like they'd apply to a specific query or context (i.e. you wouldn't want to set the same pageState for a set of queries, or have a profile that always uses prepared statements).  That being said, I'm not exactly sure if these options are worth it / or needed.